Tests/FeedStoreIntegrationTests.swift

```diff
@@ -31,60 +31,74 @@ class FeedStoreIntegrationTests: XCTestCase {
     }
     
     func test_retrieve_deliversNoEmptyOnEmptyCache() {
-//        let sut = makeSUT()
-//
-//        expect(sut, toRetrieve: .empty)
+        let sut = makeSUT()
+
+        expect(sut, toRetrieve: .empty)
     }
 
     func test_retrieve_deliversFeedInsertedOnAnotherInstance() {
-//        let storeToInsert = makeSUT()
-//        let storeToLoad = makeSUT()
-//        let feed = uniqueImageFeed()
-//        let timestamp = Date()
-//
-//        insert((feed, timestamp), to: storeToInsert)
-//
-//        expect(storeToLoad, toRetrieve: .found(feed: feed, timestamp: timestamp))
+        let storeToInsert = makeSUT()
+        let storeToLoad = makeSUT()
+        let feed = uniqueImageFeed()
+        let timestamp = Date()
+
+        insert((feed, timestamp), to: storeToInsert)
+
+        expect(storeToLoad, toRetrieve: .found(feed: feed, timestamp: timestamp))
     }
     
     func test_insert_overridesFeedInsertedOnAnotherInstance() {
-//        let storeToInsert = makeSUT()
-//        let storeToOverride = makeSUT()
-//        let storeToLoad = makeSUT()
-//
-//        insert((uniqueImageFeed(), Date()), to: storeToInsert)
-//
-//        let latestFeed = uniqueImageFeed()
-//        let latestTimestamp = Date()
-//        insert((latestFeed, latestTimestamp), to: storeToOverride)
-//
-//        expect(storeToLoad, toRetrieve: .found(feed: latestFeed, timestamp: latestTimestamp))
+        let storeToInsert = makeSUT()
+        let storeToOverride = makeSUT()
+        let storeToLoad = makeSUT()
+
+        insert((uniqueImageFeed(), Date()), to: storeToInsert)
+
+        let latestFeed = uniqueImageFeed()
+        let latestTimestamp = Date()
+        insert((latestFeed, latestTimestamp), to: storeToOverride)
+
+        expect(storeToLoad, toRetrieve: .found(feed: latestFeed, timestamp: latestTimestamp))
     }
     
     func test_delete_deletesFeedInsertedOnAnotherInstance() {
-//        let storeToInsert = makeSUT()
-//        let storeToDelete = makeSUT()
-//        let storeToLoad = makeSUT()
-//
-//        insert((uniqueImageFeed(), Date()), to: storeToInsert)
-//
-//        deleteCache(from: storeToDelete)
-//
-//        expect(storeToLoad, toRetrieve: .empty)
+        let storeToInsert = makeSUT()
+        let storeToDelete = makeSUT()
+        let storeToLoad = makeSUT()
+
+        insert((uniqueImageFeed(), Date()), to: storeToInsert)
+
+        deleteCache(from: storeToDelete)
+
+        expect(storeToLoad, toRetrieve: .empty)
     }
     
     // - MARK: Helpers
     
-    private func makeSUT() -> FeedStore {
-        fatalError("Must be implemented")
+    private func makeSUT(storeURL: URL? = nil, file: StaticString = #file, line: UInt = #line) -> FeedStore {
+        let storeBundle = Bundle(for: CoreDataFeedStore.self)
+        let sut = try! CoreDataFeedStore(storeURL: storeURL ?? testSpecificsStoreURL(), bundle: storeBundle)
```

Comment:
`storeURL` parameter is never used thus it can be removed

```suggestion
    private func makeSUT(file: StaticString = #file, line: UInt = #line) -> FeedStore {
        let storeBundle = Bundle(for: CoreDataFeedStore.self)
        let sut = try! CoreDataFeedStore(storeURL: testSpecificsStoreURL(), bundle: storeBundle)
```